### PR TITLE
Minor cleanups in examples.

### PR DESCRIPTION
**Pull request by bradh**
Mostly to silence build-time noise, but some other simplifications.

### CODE REVIEW

File: optaplanner-examples/src/main/java/org/optaplanner/examples/common/persistence/StringDataGenerator.java [hunk 1]
```diff
@@ -182,7 +182,7 @@ public StringDataGenerator addPart(String... partValues) {
         return this;
     }
 
-    /**
+    /*
```
Comment (ge0ffrey) [inline optaplanner-examples/src/main/java/org/optaplanner/examples/common/persistence/StringDataGenerator.java]: This is meant a javadoc

File: optaplanner-examples/src/main/java/org/optaplanner/examples/scrabble/domain/ScrabbleCell.java [hunk 2]
```diff
@@ -36,7 +36,7 @@
 
     @CustomShadowVariable(variableListenerClass = CellUpdatingVariableListener.class,
             sources = {@PlanningVariableReference(entityClass = ScrabbleWordAssignment.class, variableName = "startCell"),
-                    @PlanningVariableReference(entityClass = ScrabbleWordAssignment.class, variableName = "direction"),})
+                       @PlanningVariableReference(entityClass = ScrabbleWordAssignment.class, variableName = "direction")})
```
Comment (ge0ffrey) [inline optaplanner-examples/src/main/java/org/optaplanner/examples/scrabble/domain/ScrabbleCell.java]: identation is 4 spaces and twice that to against the last line

**Comment by ge0ffrey** [comment]
ok to test

File: optaplanner-examples/src/main/java/org/optaplanner/examples/scrabble/domain/ScrabbleCell.java (hunk 2 above)
Comment (bradh) [inline optaplanner-examples/src/main/java/org/optaplanner/examples/scrabble/domain/ScrabbleCell.java]: I'll remove this change, but for the future, should it be 4 for the CustomShadowVariable annotation, 8 for the `sources =` line, and 12 for the second `PlanningVariableReference` line? Or 4 / 8 / 16?

File: optaplanner-examples/src/main/java/org/optaplanner/examples/common/persistence/StringDataGenerator.java (hunk 1 above)
Comment (bradh) [inline optaplanner-examples/src/main/java/org/optaplanner/examples/common/persistence/StringDataGenerator.java]: OK. It needs to look slightly different then - the warning was for the first line to end with a full stop. Will fix.

File: optaplanner-examples/src/main/java/org/optaplanner/examples/scrabble/domain/ScrabbleCell.java (hunk 2 above)
Comment (jhrcek) [inline optaplanner-examples/src/main/java/org/optaplanner/examples/scrabble/domain/ScrabbleCell.java]: @bradh If you're using IntelliJ or Eclipse you can import these project-wide [config files](https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/tree/master/ide-configuration) and have the IDE format it for you

Comment (bradh) [inline optaplanner-examples/src/main/java/org/optaplanner/examples/scrabble/domain/ScrabbleCell.java]: @jhrcek : NetBeans?

**Comment by ge0ffrey** [comment]
jenkins please retest

**Pull request merged**